notes/2016/2016-12-15.md

```diff
@@ -0,0 +1,29 @@
+# 15-Dec-2016 ESLint TSC Meeting Notes
+
+## Transcript
+
+https://gitter.im/eslint/tsc-meetings/archives/2016/12/15
+
+## Attending
+
+* Brandon Mills (@btmills) - TSC
+* alberto (@alberto) - TSC
+* Kai Cataldo (@kaicataldo) - TSC
+* Ilya Volodin (@ilyavolodin) - TSC
+
+## Topics
+
+### [Allow `Promise.reject()` with no arguments in `no-reject-literal` rule](https://github.com/eslint/eslint/pull/7689)
+
+* TSC Summary: no-reject-literal is an accepted rule proposal that aims to prevent Promises from being rejected with non-error values. There has been considerable debate about whether the rule should have an option to allow code like Promise.reject() (i.e. calling Promise.reject without explicitly passing in an argument). On the one hand, Promise.reject() is arguably useful for control flow, so an option like this could allow a user to return a rejected Promise without triggering the rule. On the other hand, calling Promise.reject() with no arguments causes the Promise to be rejected with undefined, which is the same thing that the rule is trying to prevent.
+* TSC Question: Should we add an option to allow Promise.reject() without any explicit arguments?
+
+**Resolution**: add an option that will be off by default
+
+### Issue and PR accumulation 
+* TSC Summary: We're seeing both issues and PRs accumulate more than they have in the past. Part of the cause for PRs might be the longer blocking of merging as a result of needing a second patch release this week. Would love to discuss strategies to help mitigate this.
```

Comment:
I've also noticed that we aren't good at circling back to close issues and PRs that have lost momentum or fail to gain enough support to move forward. 

I went through and closed a bunch of such issues today. I know people get upset when issues are closed, but we need to get better at feeling comfortable doing so (for everyone's sanity).